bin/peptide_file_generation.py

```diff
@@ -134,6 +149,14 @@ def peptide_file_generation(msstats: str, sdrf: str, compress: bool, output: str
     msstats_df = pd.read_csv(msstats, sep=',', compression=compression_method)
     # Remove 0 intensity signals from the msstats file
     msstats_df = msstats_df[msstats_df[INTENSITY] > 0]
+    msstats_df[PEPTIDE_SEQUENCE] = msstats_df.apply(lambda x: sub_mod(x[PEPTIDE_SEQUENCE]), axis=1)
```

Comment:
Here, We should add a new column named PeptideCanonical, so that peptidoform are required for next step normalization. And then removing sub_modification step in peptide_normalization.py